### ISSUE TITLE
Importing from css results in `module.exports = "/_next/static/images..."`

### ISSUE THREAD

**Issue by ElForastero** [cyrilwanner/next-optimized-images#135]
Imports in CSS files produce broken images.
Not only SVG but at least JPEG too.

![image](https://user-images.githubusercontent.com/5102818/77055718-d846be00-69ea-11ea-9055-e1036b3793c8.png)

### next.config.js
```js
const withOptimizedImages = require('next-optimized-images');
require('dotenv').config();

module.exports = withOptimizedImages({
  // next-optimized-images
  handleImages: ['jpeg', 'png', 'webp', 'svg'],

  // next
  env: {
    SITE_URL: process.env.SITE_URL,
  },
  webpack: config => {
    config.resolve.modules.push(__dirname);

    return config;
  },
});
```

### Dependencies
```
 "next": "9.3.1",
 "next-optimized-images": "^2.5.5",
 "webp-loader": "^0.6.0"
 "imagemin-optipng": "^7.1.0",
 "imagemin-svgo": "^7.1.0",
```

Probbaly related to https://github.com/zeit/next.js/issues/11164

**Comment by ElForastero** [comment]
Looks like the issue is related to next.js itself. I have tried regular url-loader and result was the same.

**Comment by durchanek** [comment]
Looks like the new CSS support broke this for all similar plugins: https://github.com/zeit/next.js/issues/11164#issuecomment-602555508

> tl;dr the new CSS support does not require next-images, so next-images needs to be sure to only apply to JS files. There could be an option to "force on" for CSS files for legacy use cases.

**Comment by cyrilwanner** [comment]
Yes the cause was the newly introduced CSS support in nextjs.

But I was now finally able to overwrite the nextjs builtin image handling (only images, not the whole css/sass feature), which should hopefully solve all those errors.
Is it possible to try the canary version `2.5.8-canary.0` and check if everything works again?

**Comment by ivan-kleshnin** [comment]
Next-Images does work with new CSS modules 🤷

**Comment by cyrilwanner** [comment]
It has already been fixed within the currently published version on npm, so I'll close this issue.

Also, I'm currently working on the next major version which is a complete rewrite and should resolve all other open issues (+ providing many new features). If you are interested, you can check out the [canary branch](https://github.com/cyrilwanner/next-optimized-images/tree/canary).

**Comment by chrisLoPresti** [comment]
getting this same exact issue still :/  even after implementing changes here

**Comment by iwfan** [comment]
Encountered the same problem in nextjs v11 and next-optimized-images v2.6.2

![image](https://user-images.githubusercontent.com/28853200/126055583-24064a11-c209-461b-a429-cff3aa189efc.png)
![image](https://user-images.githubusercontent.com/28853200/126055550-402347ee-3e40-4efa-923a-54612144f263.png)

**Comment by keadex** [comment]
Same problem in Nextjs 12. I've tried also the canary release but the issue still occurs.

**Comment by keadex** [comment]
I've created a [pull request](https://github.com/cyrilwanner/next-optimized-images/pull/269) to fix the compatibility with Next.js >= 11 since the rule conditions are changed with the new Next.js release.

**Comment by wodka** [comment]
@cyrilwanner would be great to check the PR for NextJS >= 11 support
